From 5c5e4abc232786de6dd4984960071c4cfc1fecdc Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 30 Oct 2023 14:26:35 +0100
Subject: [PATCH 0023/1695] ci: add STM32MP2x platforms default build

Adds STM32MP2x platform default build to the CI.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 5b4eb1bb75959e50254d34ebce77bad16f1451b0 ci: add STM32MP2x platforms default build]
Change-Id: Ia7488e4b61b436a8b8c6828383c8c5aca35f5fa3
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
[etienne: cherry-picked from 76438b80e5689d526037fe9b80430a858f5be0ff]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 .github/workflows/ci.yml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.github/workflows/ci.yml b/.github/workflows/ci.yml
index ba0056eaa..fcfc1bc1e 100644
--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -139,6 +139,7 @@ jobs:
           _make PLATFORM=stm32mp1
           _make PLATFORM=stm32mp1-135F_DK
           if [ -d $HOME/scp-firmware ]; then _make PLATFORM=stm32mp1-157C_DK2 CFG_SCMI_SCPFW=y CFG_SCP_FIRMWARE=$HOME/scp-firmware; fi
+          _make PLATFORM=stm32mp2
           _make PLATFORM=vexpress-fvp
           _make PLATFORM=vexpress-fvp CFG_ARM64_core=y
           _make PLATFORM=vexpress-fvp CFG_ARM64_core=y CFG_CORE_SEL1_SPMC=y CFG_SECURE_PARTITION=y
-- 
2.47.2

